.expeditor/scripts/finish_release/publish_to_chocolatey.ps1

```diff
@@ -37,13 +37,20 @@ Write-Host "--- Publishing package"
 
 $valid_build_creator="Chef Expeditor"
 
-if($env:BUILDKITE_BUILD_CREATOR -eq $valid_build_creator) {
-  choco pack $tempDir/habitat.nuspec --version $Version
-  choco push habitat.$Version.nupkg -k $env:CHOCO_API_KEY --timeout 600
-} else {
-  Write-Host "--- NOT PUBLISHING: Build triggered by $env:BUILDKITE_BUILD_CREATOR and not $valid_build_creator"
-  Write-Host "choco pack $tempDir/habitat.nuspec --version $Version"
-  Write-Host "choco push habitat.$Version.nupkg -k <elided ChocoApiKey> --timeout 600"
+$pack_cmd = "choco pack $tempDir/habitat.nuspec --version $Version"
+$publish_cmd = "choco push habitat.$Version.nupkg -k $env:CHOCO_API_KEY --timeout 600"
+
+try {
+    if($env:BUILDKITE_BUILD_CREATOR -eq $valid_build_creator) {
+        Invoke-Expression $pack_cmd
+        if (-not $?) { throw "unable to choco pack" }
+        Invoke-Expression $publish_cmd
+        if (-not $?) { throw "unable to publish Chocolatey package" }
+    } else {
+        Write-Host "--- NOT PUBLISHING: Build triggered by $env:BUILDKITE_BUILD_CREATOR and not $valid_build_creator"
+        Write-Host "$pack_cmd"
+        Write-Host "$publish_cmd"
```

Comment:
This will print the API key out; before, this was explicitly elided.

Reply:
Updated to use string concatenation to use a different key string for the command execution and the NOOP output.